src/LedgerLivePlatformSDK/storage.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Storage interface to access Wallet Server store.
+ */
+export interface StorageSDK {
+  /**
+   * Store the value with the associated key.
+   * @param key - Key to find later the value
+   * @param value - Value to save
+   * @returns The false if the value has not been saved.
```

Comment:
Looks like there is a typo here.
Either use
> The value `false` if the value has not been saved.

(maybe not the best since there is a repetition of _the value_
or

> `false` if the value has not been saved.

```suggestion
   * @returns `false` if the value has not been saved.
```